src/Esfa.Vacancy.Register.Infrastructure/Caching/AzureRedisCacheService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Threading.Tasks;
+using Esfa.Vacancy.Register.Domain.Interfaces;
+using Esfa.Vacancy.Register.Infrastructure.Settings;
+using Newtonsoft.Json;
+using SFA.DAS.NLog.Logger;
+using StackExchange.Redis;
+
+namespace Esfa.Vacancy.Register.Infrastructure.Caching
+{
+
+    public class AzureRedisCacheService : ICacheService
+    {
+        private readonly Lazy<ConnectionMultiplexer> _lazyConnection;
+        private readonly ILog _logger;
+
+        public AzureRedisCacheService(IProvideSettings settings, ILog logger)
+        {
+            _logger = logger;
+            var cacheConnection = settings.GetSetting(ApplicationSettingKeyConstants.CacheConnection);
```

Comment:
Could this be called CacheConnectionString maybe?

Reply:
Could be.  I've just followed the convention of the existing RAA services so trying to keep configuration names equivalent for shared services.